plugins/org.jboss.tools.locus.dmr/1.3.0.Final/src/org/jboss/dmr/Activator.java

```diff
@@ -0,0 +1,19 @@
+/*************************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc. and others.
+ * All rights reserved. This program and the accompanying materials 
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors:
+ *     JBoss by Red Hat - Initial implementation.
+ ************************************************************************************/
+package org.jboss.dmr;
+
+/**
+ * This class is a placeholder to avoid compilation errors in this project
+ * after importing into eclipse workspace.
+ */
+public class Activator {
```

Comment:
I guess the Activator is added to get rid of "Package 'org.jboss.dmr' does not exist in this plug-in" error marker on MANIFEST.MF
But instead of adding any code to the plugin we can disable this error marker adding compilers.p.unresolved-import=1 to the .settings/org.eclipse.pde.prefs